development/image-builder/main.go

```diff
@@ -209,6 +210,17 @@ func runBuildJob(o options, vs Variants, envs map[string]string) error {
 		buildArgs = envs
 	}
 
+	if o.buildArgs != nil && len(o.buildArgs) != 0 {
```

Comment:
This line can be simplified. If this field is `nil`, then the length will always be `0`. See https://go.dev/play/p/uq-5vCVeVz6
```suggestion
	if len(o.buildArgs) > 0 {
```